Makefile-package.mk

```diff
@@ -29,7 +29,8 @@ endef
 FPM_COMMON_OPTIONS = --verbose -C tmp -s dir -n $(PROJECT_NAME) -v $(VERSION) --iteration $(RELEASE)\
  --prefix "" --license $(LICENSE) --vendor $(VENDOR) -m $(PACKAGER) --url $(PACKAGE_URL)\
  --description "$$(printf $(DESCRIPTION))"\
- --depends $(DEP_APACHE) --depends $(DEP_CASSANDRA) --depends $(DEP_MYSQL) --depends $(DEP_NGINX) --depends $(DEP_REDIS)
+ --replaces "$(DEP_APACHE)" --replaces "$(DEP_CASSANDRA)" --replaces "$(DEP_MYSQL)" --replaces "$(DEP_NGINX)" --replaces "$(DEP_REDIS)"\
+ --depends $(DEP_APACHE) --depends "$(DEP_CASSANDRA) = 2.0.0" --depends $(DEP_MYSQL) --depends $(DEP_NGINX) --depends $(DEP_REDIS)
```

Comment:
why are we fixing C* version to 2.0.0?

and if v2 is required, can't we set the depends to `>=` ?